### DISCUSSION TITLE
How to handle 2-Factor Authentication

### DISCUSSION THREAD

**Discussion by agglomerator** [vzhd1701/evernote-backup#57] [Q&A] [answered]
When I run init-db supplying the user and password using the --user and --password args, I keep getting invalid password.  I'm sure the password is correct.  My evernote login requires TFA using Google Authenticator - how do I supply the token for that in the login sequence.

**Comment by vzhd1701** [comment]
Have you tried using `--oauth` option?

**Comment by agglomerator** [comment, marked as answer]
Yes - that worked.  Could not find the --oauth option documented anywhere, may want to add that.  Thank you

**Reply by chrowe** [reply]
https://github.com/vzhd1701/evernote-backup/pull/109 is a start.
@agglomerator can you mark this thread as answered?
